src/analytics/store-analytics.js

```diff
@@ -121,8 +122,9 @@ exports.handler = async ({ request_uri_args: args }) => {
   try {
     const { startDate, endDate, cnpj, domain } = args;
     const cnpjs = [].concat(cnpj);
+    const domains = [].concat(domain);
 
-    const analysis = await askAnalysis(startDate, endDate, cnpjs, domain);
+    const analysis = await askAnalysis(startDate, endDate, cnpjs, domains);
```

Comment:
Don't you think it's too much for an API? It can take too long doesn't it?

Reply:
Nope, it's better one request with multiple domains like "mide" and "microvix" rather than multiple requests with a single domain. Did you get it? Presto is already computing data...

Reply:
okay but it is weird, and if we change the backend we will need to live with these lists of things being passed as params

Reply:
Yes. we need for now